calebwagner/cars-r-us
# added paints.js module to project

I added an array method to iterate through the paints array in the database and list out the output in a list.
Steps to check code:
1. git fetch --all
2. git checkout cjw-paint-module
3. code .
4. make sure code appears in Paint.js module
5. serve
6. make sure paints list appears in the browser
